Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
Agree.


> On Sep 11, 2015, at 3:38 PM, Kevin Bond <notifications@github.com> wrote:
> 
> In Destination/Destination.php <https://github.com/kbond/ZenstruckBackupBundle/pull/3#discussion_r39271358>:
> 
> >  
> >  /**
> >   * @author Kevin Bond <kevinbond@gmail.com>
> >   */
> > -interface Destination
> > +interface Destination extends Countable, IteratorAggregate
> What about a getBackup($key) method. getIterator should return an array keyed by the Backup key.
> 
> I'm thinking in a UI you would have a list of backups and a "delete" button. You would need an ID.
> 
> Since the key is probably a path maybe you can md5 it?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/kbond/ZenstruckBackupBundle/pull/3/files#r39271358>.
>